openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -142,105 +214,165 @@ def setUp(self):
         )
 
         # OrgA and OrgB taxonomy
-        self.tC1 = Taxonomy.objects.create(name="tC1", enabled=True)
+        self.tBA1 = Taxonomy.objects.create(name="tBA1", enabled=True)
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC1,
+            taxonomy=self.tBA1,
             org=self.orgA,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC1,
+            taxonomy=self.tBA1,
             org=self.orgB,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
-        self.tC2 = Taxonomy.objects.create(name="tC2", enabled=False)
+        self.tBA2 = Taxonomy.objects.create(name="tBA2", enabled=False)
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC2,
+            taxonomy=self.tBA2,
             org=self.orgA,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC2,
+            taxonomy=self.tBA2,
             org=self.orgB,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
 
+        # # ToDo: OrgX content library test
+        # # This will show tX1 to every studio user
+        # self.content_libraryX = ContentLibrary.objects.create(
+        #     org=self.orgX,
+        #     slug='foobar',
+        #     bundle_uuid=uuid.uuid4(),
+        #     allow_public_learning=True,
+        #     allow_public_read=True,
+        # )
+        # self.tX1 = Taxonomy.objects.create(name="tX1", enabled=True)
+        # TaxonomyOrg.objects.create(
+        #     taxonomy=self.tX1,
+        #     org=self.orgX,
+        #     rel_type=TaxonomyOrg.RelType.OWNER,
+        # )
+
```

Comment:
As [Braden noted](https://github.com/openedx/edx-platform/pull/33413#issuecomment-1760396716), you need to use [the public create_library API](https://github.com/openedx/edx-platform/blob/2033dcf6ace133719aaeb72dc5dd6ee521a7ac42/openedx/core/djangoapps/content_libraries/api.py#L409) instead of the app's internal ContentLibrary Django model.